### PR TITLE
Initial grafana2 state

### PR DESCRIPTION
**Pull request by zackhsi**
This is a state module for [Grafana v2.0](http://docs.grafana.org/guides/whats-new-in-v2/) that uses the HTTP API.

It provides a version of inheritance through pillars, in which you can specify a chain of dashboard, row or panel dictionaries to update from.

Panel spans and panel ids are smartly generated.

### CODE REVIEW

File: salt/states/grafana2.py [hunk 1]
```diff
@@ -0,0 +1,551 @@
+# -*- coding: utf-8 -*-
+'''
+Manage Grafana v2.0 Dashboards
+
```
Comment (jfindlay) [inline salt/states/grafana2.py]: @zackhsi, `.. versionadded:: Boron`

Comment (zackhsi) [inline salt/states/grafana2.py]: Done

Comment (jfindlay) [inline salt/states/grafana2.py]: Thanks, @zackhsi.

**Comment by cachedout** [comment]
Really neat! I like this a lot. Well done.

**Comment by zackhsi** [comment]
Thanks @jfindlay and @cachedout !

**Pull request merged**